config/default/kustomization.yaml

```diff
@@ -22,7 +22,7 @@ bases:
 # [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
 #- ../certmanager
 # [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
-- ../prometheus
+#- ../prometheus
```

Comment:
There's this last change I've made that is worth mentioning.

Due to the change:
```
Now operator-sdk automatically adds all generated elements in bundle. This makes include the metrics-reader Service as part of the bundle now
```

Now the `ServiceMonitor` object related to Prometheus was also automatically included in the `bundle/manifests` directory, changing the existing behavior we had using operator-sdk v1.1.0. I've commented that line to disable the automatic inclusion of the `ServiceMonitor`object in bundle and automatic installation when running `make deploy` to keep our previous behavior where the responsability of installing the `ServiceMonitor` belonged to the users when installing it through OLM.

This is specially important on clusters where Prometheus is not installed (the case in our apicast-operator where K8s native is used and Prometheus CRDs might not be installed) as including the ServiceMonitor in the bundle would make the OLM installation fail due to Prometheus API types not existing.